packages/web/docs/src/components/got-an-idea-section.tsx

```diff
@@ -0,0 +1,136 @@
+            stroke="url(#paint2_linear_2003_9879)"
+          />
+          <path
+            d="M861.052 368.164C852.38 376.837 847.5 388.608 847.5 400.864L847.5 437.5L936.24 437.5L936.24 369.868C936.24 330.858 967.858 299.24 1006.87 299.24L1431.5 299.24L1431.5 210.5L1037.86 210.5C1025.61 210.5 1013.84 215.38 1005.16 224.052L937.094 292.123L929.123 300.094L861.052 368.164Z"
+            stroke="url(#paint3_linear_2003_9879)"
+          />
+          <defs>
+            <linearGradient
+              id="paint0_linear_2003_9879"
+              x1="252.5"
+              y1="48"
+              x2="329.643"
+              y2="245.934"
+              gradientUnits="userSpaceOnUse"
+            >
+              <stop stopColor="#A2C1C4" stopOpacity="0" />
+              <stop offset="1" stopColor="#A2C1C4" stopOpacity="0.8" />
+            </linearGradient>
+            <linearGradient
+              id="paint1_linear_2003_9879"
+              x1="1139.5"
+              y1="48"
+              x2="1062.36"
+              y2="245.934"
+              gradientUnits="userSpaceOnUse"
+            >
+              <stop stopColor="#A2C1C4" stopOpacity="0" />
+              <stop offset="1" stopColor="#A2C1C4" stopOpacity="0.8" />
+            </linearGradient>
+            <linearGradient
+              id="paint2_linear_2003_9879"
+              x1="252.5"
+              y1="324"
+              x2="329.643"
+              y2="126.066"
+              gradientUnits="userSpaceOnUse"
+            >
+              <stop stopColor="#A2C1C4" stopOpacity="0" />
+              <stop offset="1" stopColor="#A2C1C4" stopOpacity="0.8" />
+            </linearGradient>
+            <linearGradient
+              id="paint3_linear_2003_9879"
+              x1="1139.5"
+              y1="324"
+              x2="1062.36"
+              y2="126.066"
+              gradientUnits="userSpaceOnUse"
+            >
+              <stop stopColor="#A2C1C4" stopOpacity="0" />
+              <stop offset="1" stopColor="#A2C1C4" stopOpacity="0.8" />
+            </linearGradient>
+          </defs>
+        </svg>
+      </DecorationIsolation>
+      <Heading as="h2" size="md" className="text-white">
+        Got an idea for a new library?
+      </Heading>
+      <p className="mt-4 text-white/80">
+        Join our community to chat with us and let's build something together!
+      </p>
+      <CallToAction
+        href="https://the-guild.dev/contact"
+        variant="primary-inverted"
+        className="mt-8"
+        onClick={event => {
+          if (window.$crisp) {
+            event.preventDefault();
+            window.$crisp?.push(['do', 'chat:open']);
+          }
+        }}
+      >
+        Get in touch
+      </CallToAction>
+    </div>
+  );
+}
+
+declare global {
+  interface Window {
+    $crisp?: {
+      push: (args: any[]) => void;
+    };
+  }
+}
```

Comment:
can you move it to `src/env.d.ts` ?

Reply:
Good idea.

Reply:
BTW I created a component for this, but it's still in v8.